.github/workflows/cspellcdict.txt

```diff
@@ -3162,3 +3182,22 @@ Gmail
 vmf
 mdl
 html
+VANTS
+Dont
```

Comment:
I'm not familiar with this - is it meant to be **don't?**?

Reply:
Inside many of the image links there is a (#DontLink) tag that the spell checker was hanging on.